Source/cielim/Private/RenderingFunctionsLibrary.cpp

```diff
@@ -26,3 +28,79 @@ void URenderingFunctionsLibrary::ApplyReadNoise()
 {
 	
 }
+
+FString URenderingFunctionsLibrary::ApplyDarkCurrentNoise(FString Filepath, double MaxSigma, double MinSigma, FVector SunPosition, FVector SpacecraftPosition, FVector SpacecraftDirection)
```

Comment:
🔶 Because this line extends over 120 characters, you can drop each parameter onto a new line.

Reply:
Put each parameter onto a new line!